.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
not sure you want to commit this
Probably want to add `.vscode` to `.gitignore`